### ISSUE TITLE
Add seed value input to the tests 

### ISSUE THREAD

**Issue by XingerTang** [AlphaGenes/AlphaPeel#170]
To fairly compare the test results.

**Comment by XingerTang** [comment]
No random function is called in both ``AlphaPeel``and relevant modules in ``tinyhouse``. 

Random seed is set in the ``tinyhouse.InputOutput``.
There are commented sections in ``tinyhouse.HaplotypeOperations`` that call random functions. It might be the reason why the seed option is included in the first place.

But the random seed now has no impact on the AlphaPeel, and therefore, the option can be removed.

**Comment by gregorgorjanc** [comment]
OK, that's "downstream", but probably best to set seed also "upstream" in simulation of the data.

**Comment by XingerTang** [comment]
> OK, that's "downstream", but probably best to set seed also "upstream" in simulation of the data.

It would be great to have a seed in the simulation data. But I didn't find the seed in the AlphaSimR manual. Is it implemented now?
We have been using the same set of simulated data for years, and I think the replicability is good so far. For future needs, it's definitely better to have seeds to control the random generator.

**Comment by gregorgorjanc** [comment]
Yes, RNG in AlphaSimR is now sorted (see https://github.com/gaynorr/AlphaSimR/blob/363ad5ca177b6306cc5d580e0b34067ddbc42dc7/NEWS.md?plain=1#L41) and will be part of 2.2 release (it has not yet been released, but if you install devel branch from GitHub then you will have this functionality). Users don't need to do anything specific from AlphaSimR side of things, they just have to use `set.seed(someNumber)` which is how we set seed globally in `R` and then AlphaSimR functions grab that.

As to our testing, I suggest you add a `set.seed(someNumber)` so we have the test simulation reproducible too - should we need it in future.

**Comment by XingerTang** [comment]
I suppose we can use current simulation data for now, and update the R code once AlphaSimR releases v2.2.
